### PR TITLE
chore(ios): Replace deprecated sdkInfo

### PR DESCRIPTION
**Pull request by krystofwoldrich**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring

## :scroll: Description
Replace deprecated sdkinfo with the new attributes

## :green_heart: How did you test it?
Sample app

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

## :crystal_ball: Next steps
#skip-changelog

### CODE REVIEW

File: ios/RNSentry.m [hunk 1]
```diff
@@ -151,8 +151,8 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
                   rejecter:(RCTPromiseRejectBlock)reject)
 {
     resolve(@{
-        @"name": PrivateSentrySDKOnly.options.sdkInfo.name,
-        @"version": PrivateSentrySDKOnly.options.sdkInfo.version
+        @"name": PrivateSentrySDKOnly.getSdkName,
+        @"version": PrivateSentrySDKOnly.getSdkVersionString
```
Comment (marandaneto) [inline ios/RNSentry.m:155]: This is a function, not sure you can do this directly.

**Comment by marandaneto** [comment]
CI is unhappy

**Comment by krystofwoldrich** [comment]
> CI is unhappy

looks like ci flakiness, now the pod install worked out

**Review by marandaneto — APPROVED** [review]

**Comment by krystofwoldrich** [comment]
Fixed https://github.com/getsentry/sentry-react-native/issues/2530

**Pull request merged**